README.md

```diff
@@ -27,26 +25,33 @@ const myFormValues = {
 }
 ```
 
-We can add a charsNotBlackList validation to the myFormValues
+We can add a charsNotBlackList validation to the myFormValues. CustomArgs are required:
 
 ```javascript
 import { charsNotBlackList } from '@lemoncode/fonk-chars-not-black-list-validator';
 
 const validationSchema = {
   field: {
-    price: [charsNotBlackList.validator],
+    product: [
+      {
+        validator: charsNotBlackList.validator,
+        customArgs: { blackListChars: 'nw5' },
+      },
+    ],
   },
 };
 ```
 
+Some characters will need to be escaped on the blacklist because they are used in a RegExp.
```

Comment:
Could you add an example related with this paragraph?

Reply:
Example added.